src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -382,6 +422,27 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         """
     }
 
+    private static String imageCreationWithInspection() {
+        """
+            import com.bmuschko.gradle.docker.tasks.image.Dockerfile
+            import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+            import com.bmuschko.gradle.docker.tasks.image.DockerInspectImage
+
+            task dockerfile(type: Dockerfile) {
+                from '$TEST_IMAGE_WITH_TAG'
+            }
+
+            task buildImage(type: DockerBuildImage) {
+                dependsOn dockerfile
+            }
+
+            task inspectImage(type: DockerInspectImage) {
+                dependsOn buildImage
+                targetImageId buildImage.getImageId()
+            }
```

Comment:
Can you add a cleanup task, which will remove the image?

Reply:
I could but non of the test cleanup so far. I was more thinking of implementing a cleanup for the whole test case.